### ISSUE TITLE
This app seems non-functional

### ISSUE THREAD

**Issue by tomrow** [LiquidFenrir/LocalFileShare#1]
The network scan finds absolutely nothing, and trying to recieve sometimes crashes the entire system.

**Comment by LiquidFenrir** [comment]
Yeah, I'm sorry was rewriting it but I never really finished it. UDS is really just too unreliable. It would be easier to make your own app using wifi and regular sockets (downside: it requires an internet connection, or figuring out how microsd management does its thing), and more reliable
